app/Models/Job.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Job extends Model
+{
+    use HasFactory;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array<int, string>
+     */
+    protected $fillable = [
+        'keywords',
+        'title',
+        'location',
+        'type'
+    ];
+
+    public function jobs() {
```

Comment:
This method should be names “employer” as the model which refers to

Reply:
Done